planning/freespace_planner/src/freespace_planner/freespace_planner_node.cpp

```diff
@@ -191,6 +191,22 @@ Trajectory createStopTrajectory(const PoseStamped & current_pose)
   return createTrajectory(current_pose, waypoints, 0.0);
 }
 
+Trajectory createStopTrajectory(const Trajectory & trajectory)
+{
+  Trajectory stop_trajectory;
+  stop_trajectory.header = trajectory.header;
+  stop_trajectory.header.stamp = rclcpp::Clock().now();
+
+  stop_trajectory.points.reserve(trajectory.points.size());
+  for (size_t i = 0; i < trajectory.points.size(); ++i) {
+    stop_trajectory.points.push_back(trajectory.points.at(i));
+  }
```

Comment:
Can these line be as follow?
```suggestion
  Trajectory stop_trajectory = trajectory;
```

Reply:
I didn't know  copy assignment can work  , I have tested it in simulation, thank u.